### PR TITLE
Add a flag to allow to schedule workload to the master node

### PR DESCRIPTION
**Pull request by yoojinl**

### CODE REVIEW

File: pkg/kubeadm/cmd/init.go [hunk 1]
```diff
@@ -77,6 +77,10 @@ func NewCmdInit(out io.Writer, s *kubeadmapi.KubeadmConfig) *cobra.Command {
 		&s.InitFlags.CloudProvider, "cloud-provider", "",
 		`(optional) enable cloud proiver features (external load-balancers, storage, etc)`,
 	)
+	cmd.PersistentFlags().BoolVar(
+		&s.InitFlags.Schedulable, "schedule-workload", false,
```
Comment (yoojinl) [inline pkg/kubeadm/cmd/init.go:81]: @lukemarsden Let me know if it's not ok to use `--schedule-workload` and I will change it to `--alsonode` as you suggested.

File: pkg/kubeadm/cmd/manual.go [hunk 2]
```diff
@@ -129,6 +129,10 @@ func NewCmdManualBootstrapInitMaster(out io.Writer, s *kubeadmapi.KubeadmConfig)
 		&s.InitFlags.Services.DNSDomain, "service-dns-domain", "cluster.local",
 		`(optional) use alterantive domain name for services, e.g. "myorg.internal"`,
 	)
+	cmd.PersistentFlags().BoolVar(
+		&s.InitFlags.Schedulable, "schedule-workload", false,
+		`(optional) allow to schedule workload to the node`,
```
Comment (yoojinl) [inline pkg/kubeadm/cmd/manual.go:134]: @errordeveloper This copy-paste does not look good, if you have some ideas how to refactor that, I can rewrite it a bit in a separate PR.

Comment (yoojinl) [reply pkg/kubeadm/cmd/manual.go:134]: E.g. the first things which comes to mind, is to extract common args into a separate method and reuse it in both `init` and `manual`.

**Comment by lukemarsden** [comment]
LGTM, thanks!

File: pkg/kubeadm/cmd/init.go (hunk 1 above)
Comment (errordeveloper) [reply pkg/kubeadm/cmd/init.go:81]: I think this is good.

**Pull request merged**